seed/models/building_file.py

```diff
@@ -278,18 +289,23 @@ def process(self, organization_id, cycle, property_view=None):
                 meter.source = m.get('source')
                 meter.type = m.get('type')
                 meter.is_virtual = m.get('is_virtual')
+                if meter.is_virtual is None:
+                    meter.is_virtual = False
                 meter.save()
 
                 # meterreadings
-                # TODO: need to check that these are in kBtu already?
+                # ENERGY_TYPES index starts at 1, tuple indexes start at 0
+                meter_type = Meter.ENERGY_TYPES[meter.type - 1][1]
+                meter_conversions = self._kbtu_thermal_conversion_factors().get(meter_type, {})
                 readings = {
                     MeterReading(
                         start_time=mr.get('start_time'),
                         end_time=mr.get('end_time'),
                         reading=mr.get('reading'),
                         source_unit=mr.get('source_unit'),
                         meter_id=meter.id,
-                        conversion_factor=1.00,  # assuming kBtu
+                        # default to a conversion_factor of 0 if the units specified are not found
+                        conversion_factor=meter_conversions.get(mr.get('source_unit'), 0.00)
```

Comment:
I think we might want to default to `1.0` -- if we do 0 then we will always get zero as a result... Not sure what the downstream effects are, but 1 seems "safe"